de.dlr.sc.virsat.swtbot.test/src/de/dlr/sc/virsat/swtbot/util/SwtBotSection.java

```diff
@@ -97,5 +99,30 @@ public Boolean run() {
 		});
 		return new SWTBotTable(table.get(0));
 	}
-
+	
+	/**
+	 * We have Section->Composite->Tree
+	 * First the method finds the composite under the section, then it finds the tree under the composite.
+	 * The general UI of the project requires that each section contains only one tree,
+	 * so we can simply return tree.get(0).
+	 * @return first tree
+	 */
+	public SWTBotTree getSWTBotTree() {

```

Comment:
I suggest you try to do it with a matcher, something like
```
return new SWTBotTree((Tree) bot.widget(WidgetMatcherFactory.widgetOfType(Tree.class), widget));
```
see http://download.eclipse.org/technology/swtbot/galileo/dev-build/apidocs/org/eclipse/swtbot/swt/finder/package-summary.html

Reply:
Thank you, thats a good idea!